frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylist.tsx

```diff
@@ -55,7 +55,7 @@ export function SessionRecordingsPlaylist({
 
     const sections: PlaylistSection[] = []
 
-    if (logicProps.logicKey) {
+    if (pinnedRecordings.length) {
```

Comment:
probing for logickey doesn't work here because it means the persons scene doesn't work

we can keep the "are there results" for now